package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "oam-server",
+  "version": "0.1.0",
+  "description": "OAM Server endpoints",
+  "main": "index.js",
+  "author": "Rob Emanuele <rdemanuele@gmail.com>",
+  "license": "Apache 2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hotosm/oam-server.git"
+  },
+  "main": "server.js",
+  "scripts": {
+    "start": "node index.js"
```

Comment:
@lossyrob what version of Node / npm are you using?  This looks like old boilerplate from `npm init`.